### ISSUE TITLE
the softmax result is differnet with pytorch nn.softmax(-1)

### ISSUE THREAD

**Issue by YLongJin** [NVIDIA/TensorRT#1713]
My input dim is 1 16 65 65,then I use the 	
nvinfer1::ISoftMaxLayer *softmax = m_network->addSoftMax(*Layers[inputName]);
softmax->setAxes(3);

but whatever the axe I use(0,1,2,3),I can not got the same result with pytorc

**Comment by ttyio** [comment]
@YLongJin , have you tried expose your pytorch model that contains only softmax only to onnx, and load the onnx in trt, is the result different?

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, please reopen if you still have issue, thanks!

**Comment by YLongJin** [comment]
Tanks  i  solove it by antoher
